corehq/apps/accounting/const.py

```diff
@@ -0,0 +1,22 @@
+DEFAULT_ACCOUNT_FORMAT = 'Account for Project %s'
```

Comment:
If these constants are specific to invoicing, we may want to rename this module to _invoice_const_ or make const a module and create an invoice.py file within that

Reply:
I didn't end up doing renaming it, because the exchange rate decimal places also seemed relevant to keep as a const (because importing constants from utils didn't seem great)

Reply:
But by renaming it, the exchange rate decimal places will be kept as invoice_const? Sorry I don't get your reasoning

Reply:
I left the file named as `const.py`, because `EXCHANGE_RATE_DECIMAL_PLACES` is not related to invoicing but it is used in multiple places, so it seemed appropriate to put into a const file.